--- gnuchess-4.0.pl80/src/configure.in~	Thu Dec  6 10:54:06 2001
+++ gnuchess-4.0.pl80/src/configure.in	Thu Dec  6 10:54:06 2001
@@ -3,18 +3,7 @@
 AC_INIT(ataks.c)
 
 dnl Checks for programs.
-if test -z "$CFLAGS" ; then
-  CFLAGS_DEFAULTED=yes
-fi
 AC_PROG_CC
-if test "$CFLAGS_DEFAULTED" = yes; then
-  if test "$GCC" = yes; then
-    CFLAGS="-O2"
-  else
-    dnl Probably every compiler has a -O flag, but not all have -O2, -O3, etc
-    CFLAGS="-O"
-  fi
-fi
 AC_PROG_INSTALL
 
 dnl Checks for libraries.
