diff -ur gnuchess-4.0.pl79.orig/src/gnuchess.h gnuchess-4.0.pl79/src/gnuchess.h
--- gnuchess-4.0.pl79.orig/src/gnuchess.h	Mon Sep 28 08:41:21 1998
+++ gnuchess-4.0.pl79/src/gnuchess.h	Tue Jul 20 19:23:18 1999
@@ -110,7 +110,7 @@
 #define printz printf
 #define scanz scanf
 #else
-#include <curses.h>
+#include <ncurses/ncurses.h>
 #define scanz fflush(stdout),scanw
 #define printz printw
 #endif
diff -ur gnuchess-4.0.pl79.orig/src/nuxdsp.c gnuchess-4.0.pl79/src/nuxdsp.c
--- gnuchess-4.0.pl79.orig/src/nuxdsp.c	Mon Sep 28 08:41:23 1998
+++ gnuchess-4.0.pl79/src/nuxdsp.c	Tue Jul 20 19:25:20 1999
@@ -38,7 +38,7 @@
 #include <sys/param.h>
 #include <sys/types.h>
 #include <sys/file.h>
-#include <curses.h>
+#include <ncurses/ncurses.h>
 
 #endif /* MSDOS */
 
diff -ur gnuchess-4.0.pl79.orig/src/uxdsp.c gnuchess-4.0.pl79/src/uxdsp.c
--- gnuchess-4.0.pl79.orig/src/uxdsp.c	Mon Sep 28 08:41:25 1998
+++ gnuchess-4.0.pl79/src/uxdsp.c	Tue Jul 20 19:24:56 1999
@@ -39,7 +39,7 @@
 #include <sys/param.h>
 #include <sys/types.h>
 #include <sys/file.h>
-#include <curses.h>
+#include <ncurses/ncurses.h>
 
 #endif /* MSDOS */
 
--- gnuchess-4.0.pl80/src/configure.in~	Thu Dec  6 11:04:35 2001
+++ gnuchess-4.0.pl80/src/configure.in	Thu Dec  6 11:04:35 2001
@@ -8,8 +8,12 @@
 
 dnl Checks for libraries.
 AC_CHECK_LIB(m, sqrt)
-AC_CHECK_LIB(termcap, tgetent)
-AC_CHECK_LIB(curses, printw)
+AC_CHECK_LIB(termcap, printw, LIBS="$LIBS -ltermcap",
+	[AC_CHECK_LIB(curses, printw, LIBS="$LIBS -lcurses",
+		[AC_CHECK_LIB(ncurses, printw, LIBS="$LIBS -lncurses")]
+	)]
+
+)
 
 dnl Checks for header files.
 AC_HEADER_STDC
