diff -ur gnuchess-4.0.pl79.orig/src/Makefile.in gnuchess-4.0.pl79/src/Makefile.in
--- gnuchess-4.0.pl79.orig/src/Makefile.in	Mon Sep 28 08:41:13 1998
+++ gnuchess-4.0.pl79/src/Makefile.in	Tue Jul 20 19:17:29 1999
@@ -127,9 +127,9 @@
 
 prefix = @prefix@
 exec_prefix = @exec_prefix@
-LIBDIR = $(prefix)/lib
+LIBDIR = $(prefix)/share/games/gnuchess
 BINDIR = $(exec_prefix)/bin
-MANDIR = $(prefix)/man/man6
+MANDIR = $(prefix)/share/man/man6
 MANEXT = .6
 
 DISTSOURCE = Makefile.ddj Makefile.dos Makefile.in Makefile.nt README ataks.c ataks.h \
